src/webviews/main.ts

```diff
@@ -90,12 +90,12 @@ export async function createVueWebview<TRequest, TResponse>(params: WebviewParam
                 "default-src 'none';
                 img-src ${view.webview.cspSource} https:;
                 script-src ${view.webview.cspSource} 'unsafe-eval';
-                style-src ${view.webview.cspSource};
+                style-src ${view.webview.cspSource} 'unsafe-inline';
```

Comment:
Slightly less secure, but since the styles and scripts are only sourced locally this is fine.
See https://csp-evaluator.withgoogle.com/